modules/assim.sequential/R/sda.enkf.R

```diff
@@ -173,12 +214,19 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
     dir.create(file.path(settings$modeloutdir, run.id[[i]]), recursive = TRUE)
     
     ## Write Configs
-    do.call(what = my.write.config, args = list(defaults = NULL, 
+    if(is.null(restart)){
+      do.call(what = my.write.config, args = list(defaults = NULL, 
                                          trait.values = params[[i]], 
                                          settings = settings, 
                                          run.id = run.id[[i]], 
                                          inputs = inputs[[i]], 
                                          IC = IC[i, ]))
+    } else {
+      ## copy over old run's forecast
```

Comment:
By old runs to you mean spin up?

Reply:
I think @mdietze refers to the previous assimilation period (or both?) but maybe he can clarify

Reply:
As per comment above, I mean a previous workflow